src/main/java/LineTokenizer.java

```diff
@@ -0,0 +1,29 @@
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import java.util.stream.Stream;
+
+public class LineTokenizer {
+    Map<String, ArrayList<Integer>> newMapWords;
+    List<Integer> linesWithWord;
+
+    public Map<String, ArrayList<Integer>> mapWords(List<String> lines) {
+        newMapWords = new HashMap<>();
```

Comment:
If you need to initialize the `newMapWords`, please do it in the constructor.

Reply:
fixed